Week1/P5 SOLUTION.py

```diff
@@ -0,0 +1,13 @@
+A = input('Enter the string :')
+c = 0
+
+A = A.lower()
+for i in A:
+    if i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u':

```

Comment:
also include capital latters too like i=='A' or i=='E' or i=='I' or i=='O' or i=='U'.

Reply:
thank you

Reply:
Welcome.